### DISCUSSION TITLE
1.7.X and typescript + web components issue

### DISCUSSION THREAD

**Discussion by codylindley** [solidjs/solid#1671] [General]
Anyone else seeing this issue when using a non-defined custom html element in JSX:

https://playground.solidjs.com/anonymous/51824c1b-da8c-4343-b52d-b5160259b9d8

![image](https://user-images.githubusercontent.com/115456/229595945-e4b2fece-287e-485f-93ca-fba626fef473.png)

**Comment by codylindley** [comment]
Still no dice: https://playground.solidjs.com/anonymous/83617ff0-014c-493b-b36b-b7266d470280

**Comment by codylindley** [comment]
Using solid-element: https://stackblitz.com/edit/solidjs-templates-9bmkwx?file=src%2Findex.tsx

**Comment by ryansolid** [comment]
I don't see the first issue. The second playground is trying to use a directive without a javascript compatible name so it is causing an error. The 3rd example in stackblitz worked for me after removing the node modules and the package lock and re-installing. Seems to be just an older version of our babel plugin.

**Comment by codylindley** [comment]
Thank you for a sanity check. So, odd, that the playground is now showing no error for me as well. Yesterday, I only got the error shown in the initial image. As well, we are seeing this in production apps that use a custom html element in jsx after upgrading to 1.7.X. I'll keep digging to see if we can find the issue.
